paasta_tools/utils.py

```diff
@@ -2067,7 +2067,7 @@ def get_soa_cluster_deploy_files(
 
     for yaml_file in glob.glob('%s/*.yaml' % service_path):
         try:
-            with open(yaml_file) as fh:
+            with open(yaml_file):
                 cluster_re_match = re.search(search_re, yaml_file)
```

Comment:
wait what? We are not even using fh? Why are we even opening it??

Reply:
This is a check to identify a real file (ie skips dead/bogus symlinks. :)